hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/ContractResultServiceImpl.java

```diff
@@ -249,6 +252,21 @@ private void processContractResult(
         entityListener.onContractResult(contractResult);
     }
 
+    private void processContractNonce(ContractFunctionResult functionResult) {
+        var nonceInfos = functionResult.getContractNoncesList();
+        if (!nonceInfos.isEmpty()) {
+            nonceInfos.forEach(nonceInfo -> {
```

Comment:
nit: Could be more succinct.
```suggestion
        functionResult.getContractNoncesList().forEach(nonceInfo -> {
```